src/gc/gcee.cpp

```diff
@@ -687,6 +687,15 @@ bool GCHeap::RuntimeStructuresValid()
     return GCScan::GetGcRuntimeStructuresValid();
 }
 
+void GCHeap::ControlEvents(GCEventKeyword keyword, GCEventLevel level)
+{
+    GCEventStatus::Set(GCEventProvider_Default, keyword, level);
+}
+
+void GCHeap::ControlPrivateEvents(GCEventKeyword keyword, GCEventLevel level)
```

Comment:
I realize that I missed the end of the review for the design so this probably came up then.  Is there a reason to have two functions here for controlling events instead of just one that takes the provider info?

Reply:
no particular reason other than not exposing `GCEventProvider` across the interface. I don't mind either way!

Reply:
Ok, I'm fine either way as well.  I doubt we'd add more providers - likely we'd reduce to just one (if that ever happens).